sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -64,7 +64,7 @@ grammar SqlBase;
 }
 
 singleStatement
-    : statement EOF
+    : statement (';')*? EOF
```

Comment:
should a single statement support multiple `;`?

Reply:
@cloud-fan I would expect the most common scenario would be the user unintentionally submitted an extra semicolon. Do you think the query should fail in that case or is the intention so obvious that it's essentially explicit? In cases where multiple statements are allowed i.e. spark-sql, would the additional semicolons just equate to empty statements that are ignored or would something else happen that could have an effect on performance?

Reply:
seems fine.

BTW should `';'*` be good enough? `*` means 0 or more occurrence.

Reply:
Yes @cloud-fan you're right, I updated this and also squashed all the commits.